### DISCUSSION TITLE
سرور ایران برای تانل

### DISCUSSION THREAD

**Discussion by iamfarhad** [hiddify/Hiddify-Manager#761] [Q&A سوالات رایج]
سلام

من سرور ایران میگیرم و با GOST به سرور خارجم تانل میکنم.
بعد از چند ساعت IP خارجم توی اون دیتاسنتر فیلتر میشه و تانل قطع میشه.

چه کارباید کرد که اینطوری نشه؟ از کجا سرور مانسب برای ترافیک بالا بگیرم؟

**Comment by omid-j-d** [comment]
سلام
دقیق بخوام بگم مشکل از روش تانلیه که انجام میدین
از forward+tls و پورت ۴۴۳ استفاده کنید همچنین ssl فراموش نشه

**Reply by alila23** [reply]
میشه لطفا دقیق تر بگید

**Comment by adljooy** [comment]
اگه مشکل ترافیک دارین [سرور مجازی ایران نوین هاست](https://www.novinhost.org/vps/iran-novin/) بدون محدودیت ترافیک ارائه میشه ، تیکت بزنین براتون انجام میدن.

**Reply by ArsamAli44** [reply]
ثبت نام بدون احراز هویت هستش؟ خب از کجا بدونیم که پشتیبان شنا هستین

**Comment by cloner9721188** [comment]
سرور ایران زدم برای تانل ولی Https نیست و کانفیگ وارد برنامه نمیشه

**Comment by Ramtisin1387** [comment]
سرور مجازی ایران نامحدود یک سرور خوب برای تانل مدنظرمه چی پیشنهاد میدزد

**Comment by pa3697** [comment]
درود به همگی دوستان 
دوستان کسی می دونه از کدوم ISP میشه VPS ایران رو گرفت که بهش بتونه SSH زد چه از خارج چه از ایران برای Tunneling ممنون میشم اگه کسی می دونه راهنمایی کنه

**Comment by suji021** [comment]
دوستان آخرین آپدیت لطفا بدید برای تانل و ریورساید تانل زدن من از آروان سرور گرفتم و عملا هیچ نتی نداره و با mirror آپدیت زدم و بعد آفلاین همه چیز رو نصب کردم )جر خوردم( و باز تونل کار نمیده 
سرویس مناسب و جواب کسی میدونه؟

**Reply by Samanshahed** [reply]
منم مثل خودتم اروان بسته همه چیاشو

**Comment by sinamokhtari534** [comment]
دوستان اگه vps داخلی که بشه از خارج هم بهش متصل شد پیدا کردید لطفا بگید
